support/langkit_support-lexical_envs.ads

```diff
@@ -49,6 +49,10 @@ package Langkit_Support.Lexical_Envs is
    --  ``Disabled`` means no caching will happen.
 
    Debug_Mode : constant Boolean := True;
+   --  Static constant used to trigger traces at compile time. TODO ??? This is
+   --  never used in practice, and profiling on big Ada codebases shows that it
+   --  has absolutely no effect on execution time, so we should probably remove
+   --  it.
```

Comment:
This looks trivial to just remove: have you tried to do it? (not clear reading the comment)